public/js/src/bridge/bridge_command.tag

```diff
@@ -458,6 +459,20 @@
         }
     }
 
+    getDataDogLink(evt) {
+
+        if (!self.shipment || !self.shipment.containers) {
+            return;
+        }
+
+        var type = getLbType(self.shipment.containers);
+        if (type === 'alb' || type === 'ingress') {
```

Comment:
lbtype for ingress currently is alb-ingress to distinguish from possible future other ingress controllers...